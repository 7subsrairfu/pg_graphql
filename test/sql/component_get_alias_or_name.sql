select graphql.alias_or_name($$
    {
        "kind": "Field",
        "name": {"kind": "Name", "value": "hello"},
        "alias": {"kind": "Name", "value": "hello_alias"},
        "arguments": null,
        "directives": null,
        "selectionSet": null
    }
$$::jsonb);


select graphql.alias_or_name($$
    {
        "kind": "Field",
        "name": {"kind": "Name", "value": "hello"},
        "alias": null,
        "arguments": null,
        "directives": null,
        "selectionSet": null
    }
$$::jsonb);
