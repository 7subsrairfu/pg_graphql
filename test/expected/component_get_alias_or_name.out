select graphql.alias_or_name_literal($$
    {
        "kind": "Field",
        "name": {"kind": "Name", "value": "hello"},
        "alias": {"kind": "Name", "value": "hello_alias"},
        "arguments": null,
        "directives": null,
        "selectionSet": null
    }
$$::jsonb);
 alias_or_name_literal 
-----------------------
 hello_alias
(1 row)

select graphql.alias_or_name_literal($$
    {
        "kind": "Field",
        "name": {"kind": "Name", "value": "hello"},
        "alias": null,
        "arguments": null,
        "directives": null,
        "selectionSet": null
    }
$$::jsonb);
 alias_or_name_literal 
-----------------------
 hello
(1 row)

